### PR TITLE
v3.2.0

### PR DESCRIPTION
**Pull request by jotathebest**
Removing support for Particle Mesh IoT Protocol

### CODE REVIEW

File: src/UbiBuilder.h [hunk 1]
```diff
@@ -25,10 +25,15 @@ Developed and maintained by Jose Garcia for IoT Services Inc
 #define _UbiBuilder_H_
 
 #include <map>
+
```
Comment (crisap94) [inline src/UbiBuilder.h:28]: There are many includes here that are not supposed to be. 
Check which of them are necessary.

**Review by crisap94 — CHANGES_REQUESTED** [review]
Take a look at the comments.

File: src/UbiHttp.h [hunk 2]
```diff
@@ -24,16 +24,15 @@ Developed and maintained by Jose Garcia for IoT Services Inc
 #ifndef _UbiHttp_H_
 #define _UbiHttp_H_
 
+#include "UbiConstants.h"
```
Comment (woakas) [inline src/UbiHttp.h:27]: It seems that this include is not used.

File: src/UbiTcp.h [hunk 3]
```diff
@@ -24,16 +24,15 @@ Developed and maintained by Jose Garcia for IoT Services Inc
 #ifndef _UbiTcp_H_
 #define _UbiTcp_H_
 
+#include "UbiConstants.h"
```
Comment (woakas) [inline src/UbiTcp.h:27]: It seems that this include is not used

Comment (jotathebest) [reply src/UbiTcp.h:27]: it is used by UbiHttp.cpp, which import this header

File: src/UbiHttp.h (hunk 2 above)
Comment (jotathebest) [reply src/UbiHttp.h:27]: it is used by UbiHttp.cpp, which import this header

**Review by crisap94 — APPROVED** [review]

**Pull request merged**